### PR TITLE
nic_hotplug: Add a new nic model igb

### PR DESCRIPTION
**Pull request by yanglei-rh**
For the igb device add hotplug/unplug scenarios.

ID:1546
Signed-off-by: Lei Yang leiyang@redhat.com

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Test Pass:
 (1/6) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.igb.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.nic_hotplug.one_pci.nic_igb.q35: PASS (258.79 s)
 (2/6) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.igb.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.nic_hotplug.multi_pci.nic_igb.q35: PASS (408.03 s)
 (3/6) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.igb.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.nic_hotplug.additional.nic_igb.q35: PASS (156.42 s)
 (6/6) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.igb.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.nic_hotplug.used_netdev.nic_igb.q35: PASS (101.33 s)

File: qemu/tests/cfg/nic_hotplug.cfg [hunk 1]
```diff
@@ -61,6 +61,11 @@
         - nic_e1000e:
             only i386, x86_64
             pci_model = e1000e
+        - nic_igb:
+            only q35
+            only x86_64
+            required_qemu = [8.1.0, )
```
Comment (PaulYuuu) [inline qemu/tests/cfg/nic_hotplug.cfg:67]: Need to change it like mentioned in VT.

**Review by PaulYuuu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
Hello @yanglei-rh, does this support Windows guest also?

**Comment by yanglei-rh** [comment]
> Hello @yanglei-rh, does this support Windows guest also?

Yes, it also can support windows guest. I tried test these scenarios with windows guest, they are also can test pass.

**Pull request merged**